dev/deps/spark-deps-hadoop-2.7

```diff
@@ -17,6 +17,7 @@ arpack_combined_all-0.1.jar
 arrow-format-0.12.0.jar
 arrow-memory-0.12.0.jar
 arrow-vector-0.12.0.jar
+audience-annotations-0.7.0.jar
```

Comment:
Is it required at Spark? Otherwise, can we exclude this in Parquet 1.11.0 RC?

Reply:
@dongjoon-hyun it is a new dependency for Parquet to annotate which API is public and what is private. I don't think that we can exclude it from Parquet, since it is used there. The annotations are Yetus audience annotations with runtime retention policy (I think class retention policy would be more adequate), therefore I'm afraid the had to be present at runtime.